### ISSUE TITLE
MultiSelect - Accessibility Improvements

### ISSUE THREAD

**Issue by sane6** [openvi-foundation/openvue#170]
### Describe the bug

The MultiSelect component has multiple accessibility violations. Please refer to this Storybook example, where the violations are listed in the Accessibility tab: https://semsveta.github.io/storybook-issue/?path=/story/multiselect--default

### Reason for not contributing a PR

- [x] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/primevue-4-vite-issue-template-jcmtragd?file=src%2FApp.vue

### Environment

Accessibility Testing Tool: 
Example: axe-core, Storybook Accessibility Addon

### Vue version

3.5.17

### PrimeVue version

4.3.6

### Node version

20.18.0

### Browser(s)

Chrome 138.0.7204.169

### Steps to reproduce the behavior

1. Please go to git pages: https://semsveta.github.io/storybook-issue/?path=/story/multiselect--default
2. Check the Accessibility tab to see issues listed

### Expected behavior

Expected behavior: at least critical violations 'Form label' : 'aria-label attribute does not exist or is empty'.
'aria-labelledby attribute does not exist' should not be present.

Copied to track in our fork; not affiliated with the original report being closed/resolved upstream

Originally reported at: [primefaces#8010](https://github.com/primefaces/primevue/issues/8010)
Originally reported by: https://github.com/Semsveta
